src/Nethermind/Nethermind.Blockchain/Nethermind.Blockchain.csproj

```diff
@@ -2,7 +2,8 @@
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
     <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
-    <LangVersion>8.0</LangVersion>
+    <LangVersion>9.0</LangVersion>
+    <Nullable>enable</Nullable>
```

Comment:
This introduces a whole load of warnings in some projects, but they are pretty easy to resolve and it gives much better static analysis features once someone goes through the effort of resolving the warnings.

Reply:
exactly - see my comment before, 30 hours of work, some of it risky, love this kind of work, very relaxing, but have no time for it
have done 20 hours of it in February and cleanu up 5 - 10 projects at that time